package.json

```diff
@@ -7,11 +7,15 @@
     "webtreemap": "build/src/cli.js"
   },
   "dependencies": {
-    "commander": "^2.11.0"
+    "commander": "^2.11.0",
+    "open": "^7.2.1",
+    "tmp": "^0.2.1"
```

Comment:
Hrrmm.  We run this in Google and probably don't have these packages.

What's the alternative?  Can you make this an optional dependency?  (E.g. only import if opening in a browser?)

Reply:
Oh wow, I can only imagine how `//third_party/node_modules` works...

- For `tmp`, I can write something custom using [`os.tmpdir`](https://nodejs.org/api/os.html#os_os_tmpdir) to sever the dependency.
- For `open` I can switch to a dynamic import.